### DISCUSSION TITLE
How to show achievements?

### DISCUSSION THREAD

**Discussion by Gui88** [community/community#40543] [New to GitHub] [answered]
Hi, I want to show my achievements on my profile page but they are not there. I have the checkbox on the settings enabled. 
![image](https://user-images.githubusercontent.com/16513472/205042963-7706a1f1-0107-4e3f-bc92-2f7439718dba.png)

I tried turning it off and on again and didn't work. Is there anything else I need to do? Thanks

**Comment by DSDmark** [comment]
Hi @Gui88,

Did you update it?

![image](https://user-images.githubusercontent.com/85744816/205056157-01a6de28-927a-4f9d-8929-cf4965f92900.png)

**Reply by Gui88** [reply]
Yes i did. It was on by default.

**Reply by DSDmark** [reply]
@Gui88, It's not default, but if you did that, then I think you haven't any achievements. If it is not case. Then /can you add screenshot here. If you can see your achievements.

**Comment by jfullstackdev** [comment, marked as answer]
seems you did not achieve anything,

![image](https://user-images.githubusercontent.com/47092464/205063949-85c989e4-71c7-48a5-a947-c6a0e5c67538.png)

try checking it here

https://github.com/Gui88?tab=achievements

because what I'm seeing is 404, you should be seeing anything when logged in, if there is none, you did not achieve anything

it looks like this 

![image](https://user-images.githubusercontent.com/47092464/205064516-104791cd-722e-4ae1-bfe6-b36b0b33286e.png)
